open_alchemy/build/__init__.py

```diff
@@ -23,6 +32,29 @@
     _INIT_TEMPLATE = in_file.read()
 
 
+class PackageFormat(Flag):
+    """Define the available package formats for the build."""
+
+    NONE = auto()
+    SDIST = auto()
+    WHEEL = auto()
+
+
+def validate_dist_format(format_) -> None:
+    """
+    Ensure a valid package format is provided.
+
+    Raises a ``BuildError` exception if the format is invalid.
+
+    :param PackageFormat format_: the package format to validate
```

Comment:
There is a pattern for these comments, please follow it. Take a look at the docstyle for `run` below for example :smile:

Reply:
Got it...  (sorry I am more used to the other way to write docstrings. Old habits die hard 😅 )